pbp.sh

```diff
@@ -1,27 +1,14 @@
-echo "$(tput setaf 5)Pulling and bundling consumer-layouts...$(tput sgr0)"
-cd consumer-layouts
-git pull
-bundle install
+#!/usr/bin/env bash
```

Comment:
Hey Colin, what does this do, exactly? Just enabled you to run with `./pbp.sh` instead of having to do `sh pbp.sh`, yeah?